### DISCUSSION TITLE
Restart full network

### DISCUSSION THREAD

**Discussion by Matssa56** [Vaskivskyi/ha-asusrouter#788] [Q&A]
Hey everyone, I had issues with my router and wifi so I reset everything from scratch. I also have a node in aimesh.

When rebooting the router, I see that the node doesn't seem to reboot, so the devices changes to the node and doesn't really come back to the router. Is there a way to reboot the router and the nodes? In Asus website you need to go under the aimesh parameters to reboot the full system. You can do the same through their app as well and reboots everything automatically if aimesh detected.

Thanks!

**Comment by Vaskivskyi** [comment]
Hello. There is no such possibility yet in the AsusRouter integration. But it can be potentially added. Please, open a corresponding [feature request](https://github.com/Vaskivskyi/ha-asusrouter/issues/new/choose), so that we can discuss details and I can start working on it

**Reply by Matssa56** [reply]
#793 opened :-)
